src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -939,6 +939,28 @@ void uct_ib_mlx5_destroy_qp(uct_ib_mlx5_md_t *md, uct_ib_mlx5_qp_t *qp)
     }
 }
 
+size_t uct_ib_mlx5_devx_sq_length(size_t tx_qp_length)
+{
+    return ucs_roundup_pow2_or0(tx_qp_length * UCT_IB_MLX5_MAX_BB);
+}
+
+size_t uct_ib_mlx5_sq_length(const uct_ib_mlx5_md_t *md, size_t tx_qp_length,
```

Comment:
1. why do we need common function - seems in every place we use it, we already know the QP type
2. maybe it should be called uct_ib_mlx5_tx_cq_length?